tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -436,8 +420,6 @@ else if (args[i].equals("-L") || args[i].equals("-l"))
            doLevelChecking    = !doLevelChecking;
       else if (args[i].equals("-D") || args[i].equals("-d")) 
            doDebugging        = !doDebugging;
-      else if (args[i].equals("-STAT") || args[i].equals("-stat")) 
```

Comment:
Please continue to accept the parameter. While it can be ignored, we should avoid breaking any existing scripts that may still pass this parameter, regardless of the reason.